README.md

```diff
@@ -4,10 +4,12 @@ Welcome to PayPal's iOS SDK. This library will help you accept card, PayPal, Ven
 
 ## FAQ
 ### Availability
-The SDK is currently in the development process. This product is being developed fully open source - throughout the development process, we welcome any and all feedback. Aspects of the SDK _will likely_ change as we develop the SDK. We recommend using the SDK in the sandbox environment until an official release is available. This README will be updated with an official release date once it is generally available.
+This product is fully open source. We welcome any and all feedback.
```

Comment:
I think this sentence can move into the `Contribution` section, and when we can delete the `Availability` section